src/views/self/view_self.cpp

```diff
@@ -163,6 +167,30 @@ namespace big
 				ImGui::EndCombo();
 			}
 		});
+		ImGui::NewLine();
```

Comment:
What does `ImGui::NewLine();` even do?
ImGui does adds elements an the next line automatically so why do this?

Reply:
Undo a SameLine() or force a new line when in a horizontal-layout context. Should probably be removed.